Dnn.CommunityForums/class/Utilities.cs

```diff
@@ -410,12 +341,6 @@ public static string NavigateUrl(int tabId, string controlKey, string pageName,
             s = Common.Globals.FriendlyUrl(ti, sURL, pageName, portalSettings);
             return s;
         }
-
-        public static void BindEnum(DropDownList pDDL, Type enumType, string pColValue, bool addEmptyValue)
```

Comment:
This is removed... Is there a replacement?  Since it's public, shouldn't we follow a deprecation policy?

Reply:
The aren't removed altogether---just in that code file. I moved them to a different code file Deprecated/Utilities.cs, and marked them as deprecated in future release.